app/controllers/metrics_controller.rb

```diff
@@ -33,10 +33,25 @@ def get_watchlist_instances
 		# On error, a 404 error is returned
 		begin
 			course_name = params[:course_name]
-			conditions = WatchlistInstance.get_instances_for_course(course_name)
-			render json: conditions
+			instances = WatchlistInstance.get_instances_for_course(course_name)
+			render json: instances
```

Comment:
To be consistent across all endpoints, should we also add status: :ok here as well? If necessary, I'll make sure I add these for subsequent endpoints too.

Reply:
Yes, I agree that we should return status =>:ok. I will amend that as well